packages/chain-bridge/src/adapters/StargateAdapter.sol

```diff
@@ -848,6 +1304,18 @@ contract StargateAdapter is Ownable, IBridgeAdapter, ILayerZeroComposer {
         }
     }
 
+    /**
+     * @dev Check if an address is a FleetCommander by checking if it has the required interface
+     * @dev Simple interface check for FleetCommander functionality
+     */
+    function _isFleetCommander(address recipient) internal view returns (bool) {
+        try IFleetCommanderMinimal(recipient).asset() returns (address) {
```

Comment:
fleet commanders are listed in the harbor command contract.

from Admirals Quarters:

```ts
    function _validateFleetCommander(address fleetCommander) internal view {
        if (
            !IHarborCommand(harborCommand()).activeFleetCommanders(
                fleetCommander
            )
        ) {
            revert InvalidFleetCommander();
        }
    }
```